src/sentry/search/snuba/executors.py

```diff
@@ -1130,6 +1130,24 @@ class InvalidQueryForExecutor(Exception):
 
 
 class GroupAttributesPostgresSnubaQueryExecutor(PostgresSnubaQueryExecutor):
+    def get_last_seen_filter(self, search_filter: SearchFilter, joined_entity: Entity) -> Condition:
+        # get the max timestamp of the error/search_issue event
```

Comment:
This doesn't really get the max does it? Just filters to timestamps before that period.

Does this work correctly for last seen? Last seen should be the aggregated max of each group, not just the timestamp of events.

Reply:
@wedamija hmm...good point. Need to think about how to restructure this because we can't use an aggregation in a where clause.

Reply:
Yeah, it can be in having, so we'd need some switch logic to enable that. We use that aggregate logic for sorting... at the very least, we filter on it during pagination https://github.com/getsentry/sentry/blob/aa29dbf1bb5afec939cb73e138228ce5f59efb5b/src/sentry/search/snuba/executors.py#L348-L349

Not sure if there are actual user accessible queries that use the aggregates